src/main/java/com/sintern/service/OpenInternPositionService.java

```diff
@@ -1,58 +1,19 @@
 package com.sintern.service;
 
+import com.sintern.domain.dto.OpenInternPositionDTO;
 import com.sintern.domain.entity.Domain;
-import com.sintern.domain.enums.DomainType;
 import com.sintern.domain.entity.OpenInternPosition;
-import com.sintern.domain.dto.OpenInternPositionDTO;
-import com.sintern.repository.OpenInternPositionRepository;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Service;
+import com.sintern.domain.enums.DomainType;
 
-import java.util.ArrayList;
 import java.util.List;
-import java.util.stream.Collectors;
-
-@Service
-public class OpenInternPositionService {
-    private final OpenInternPositionRepository openInternPositionRepository;
```

Comment:
Not saying this is wrong, but it's **out-of-scope** for this story. For such a refactor, we would have a separate **fix** branch. We try to keep only the relevant code for the story we're working on.

Reply:
You leave the way it is and ping me.